pyproject.toml

```diff
@@ -25,10 +25,8 @@ django-babel = "^0"
 django-bootstrap4 = "~1"
```

Comment:
Where do we use django-bootstrap? In emails?
Probably django recaptcha can be dropped too?

Reply:
Right now it's still used in payment gateways (braintree and dummy templates). I will removed that in #5030